### ISSUE TITLE
Demonstrate building a native Android project

### ISSUE THREAD

**Issue by big-guy** [gradle/gradle-native#611]

**Comment by trevjonez** [comment]
I assume this is blocked by #169?

**Comment by lacasseio** [comment]
Not exactly, it's possible now, it's just nowhere near what we may want users to do. The basic idea of those demonstrations is to see where we should invest our engineering effort to bridge the gap instead of moving on features that have little to no impact on what users can do.

**Comment by 9468305** [comment]
Any progress on this?

**Comment by lacasseio** [comment]
No, and if you want I can provide you some guidance for writing the sample. It should be possible to accomplish with what is already available.

**Comment by 9468305** [comment]
It's greatly appreciated.
I have a small native library project used for Android and iOS, which only have 2 c files and 1 JNI c files. 
I use standard Android Studio Gradle/CMake workflow, gradle assemble/build and output aar files(with multiple arch so files inside) for Android.
And use another [cmake-toolchains](https://cmake.org/cmake/help/v3.15/manual/cmake-toolchains.7.html#cross-compiling-for-ios-tvos-or-watchos) workflow, build `SHARED` or `STATIC` library as Framework Bundle for iOS device/simulator.

I'm really looking forward to seeing the unified gradle workflow.

**Comment by lacasseio** [comment]
The other alternative is to wrap the CMake build in Gradle so you keep using the tools offered by Android NDK. It depends the end goal. We have a [CMake wrapping sample](https://github.com/gradle/native-samples/tree/master/cpp/cmake-library) already.
